components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/AddonsManagerAdapter.kt

```diff
@@ -311,7 +313,14 @@ class AddonsManagerAdapter(
         // Add recommended section and addons if available
         if (recommendedAddons.isNotEmpty()) {
             itemsWithSections.add(Section(R.string.mozac_feature_addons_recommended_section, true))
-            itemsWithSections.addAll(recommendedAddons)
+            if (excludedAddons.isNotEmpty()) {
```

Comment:
nit: We can actually remove this if statement now as you can always filter. If the list is empty (default) all add-ons will be kept and this also should not make a performance difference.

Reply:
(But definitely OK to keep it as-s if you prefer it this way)